dags/process_elife_zip_dag.py

```diff
@@ -197,14 +197,28 @@ def wrap_article_in_libero_xml_and_send_to_service(**context) -> None:
         root
     )
 
+    return etree.tostring(xml, xml_declaration=True, encoding='UTF-8')
+
+
+def send_article_to_content_service(**context) -> None:
+    libero_xml = get_article_from_previous_task(context)
+
+    # get article id
+    xpath = '//libero:item/jats:article/jats:front/jats:article-meta/jats:article-id[@pub-id-type="publisher-id"]'
+    namespaces = {'libero': 'http://libero.pub', 'jats': 'http://jats.nlm.nih.gov'}
+    article_id = libero_xml.xpath(xpath, namespaces=namespaces)[0].text
+
     # make PUT request to service
     response = requests.put(
         '%s/items/%s/versions/1' % (SERVICE_URL, article_id),
-        data=etree.tostring(xml, xml_declaration=True, encoding='UTF-8'),
+        data=etree.tostring(libero_xml, xml_declaration=True, encoding='UTF-8'),
         headers={'content-type': 'application/xml'}
     )
     response.raise_for_status()
-    logger.info('RESPONSE= %s: %s', response.status_code, response.text)
+    logger.info('Libero wrapped article %s sent to %s with status code %s',
+                article_id,
+                SERVICE_URL,
+                response.status_code)
```

Comment:
would keep the `response.text`; it's probably not helpful now but it could be in general e.g. being a [problem detail](https://tools.ietf.org/html/rfc7807) response

Reply:
the text can be seen in the logs for the previous task. This task is simply taking that text and sending it to the content store.

Reply:
I'm not sure what the API returns as a response body, so ok with not logging possible garbage for now

Reply:
response body returns the data that was sent at the moment.